plugins/inventory/zabbix_inventory.py

```diff
@@ -113,6 +113,15 @@
                     - Will return hosts that are linked to the given proxies.
                     - Wildcard search is possible.
                     - Case-sensitive search.
+            proxy_group:
```

Comment:
proxy group filter as well as proxy filter works with names, when output we get using dynamic inventory has `zabbix_proxyid` and `zabbix_proxy_groupid`.
Maybe its worth having names in the output also for the compatibility with host_module.

Reply:
Added `resolve_id_to_names()` function. This function resolves IDs to names and adds them to the output as additional parameters. If a name for any ID is not found among the available ones, it will be requested from Zabbix. 

Currently, the following parameters are supported:
    - `proxyid` -> `proxy_name`
    - `proxy_groupid` -> `proxy_group_name`

Example of output:

```json
{
    "zabbix_name": "test-host",
    "zabbix_proxy_group_name": "inventory test proxy group",
    "zabbix_proxy_groupid": "5",
    "zabbix_proxy_name": "",
    "zabbix_proxyid": "0",
}
```

```json
{
    "zabbix_name": "test-host-2",
    "zabbix_proxy_group_name": "",
    "zabbix_proxy_groupid": "0",
    "zabbix_proxy_name": "inventory test proxy",
    "zabbix_proxyid": "5",
}
```